presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/windowSpecificationBcpAA.sql

```diff
@@ -0,0 +1,8 @@
+-- database: presto; groups: window;
+select
+suppkey, orderkey, partkey,
+nth_value(quantity, 4) over (partition by partkey order by orderkey rows between UNBOUNDED preceding and CURRENT ROW) nth_value_quantity_B,
+sum(quantity) over (partition by suppkey order by orderkey rows between unbounded preceding and current row) sum_quantity_A,
+sum(discount) over (partition by suppkey order by orderkey rows between unbounded preceding and current row) sum_discount_A
```

Comment:
Does window function can be only placed in select list of the query? Can you put a window function into an ORDER BY (HAVING or GROUP BY) clause? Will such window merge together?